[2025-09-13T17:55:22.986+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-13T17:55:23.072+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: currency_etl.analyze manual__2025-09-13T10:27:29.902252+00:00 [queued]>
[2025-09-13T17:55:23.091+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: currency_etl.analyze manual__2025-09-13T10:27:29.902252+00:00 [queued]>
[2025-09-13T17:55:23.092+0000] {taskinstance.py:2865} INFO - Starting attempt 4 of 5
[2025-09-13T17:55:23.121+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): analyze> on 2025-09-13 10:27:29.902252+00:00
[2025-09-13T17:55:23.148+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'currency_etl', 'analyze', 'manual__2025-09-13T10:27:29.902252+00:00', '--job-id', '247', '--raw', '--subdir', 'DAGS_FOLDER/currency_etl.py', '--cfg-path', '/tmp/tmplt2gdh5k']
[2025-09-13T17:55:23.155+0000] {standard_task_runner.py:105} INFO - Job 247: Subtask analyze
[2025-09-13T17:55:23.156+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1219) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-13T17:55:23.158+0000] {standard_task_runner.py:72} INFO - Started process 1220 to run task
[2025-09-13T17:55:24.366+0000] {task_command.py:467} INFO - Running <TaskInstance: currency_etl.analyze manual__2025-09-13T10:27:29.902252+00:00 [running]> on host 6eb00eb09733
[2025-09-13T17:55:24.854+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='currency_etl' AIRFLOW_CTX_TASK_ID='analyze' AIRFLOW_CTX_EXECUTION_DATE='2025-09-13T10:27:29.902252+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-13T10:27:29.902252+00:00'
[2025-09-13T17:55:24.865+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-13T17:55:24.895+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-13T17:55:24.898+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/scripts/analyze.py']
[2025-09-13T17:55:24.927+0000] {subprocess.py:86} INFO - Output:
[2025-09-13T17:55:27.281+0000] {subprocess.py:93} INFO - /opt/***/scripts/analyze.py:157: DeprecationWarning: datetime.datetime.utcnow() is deprecated and scheduled for removal in a future version. Use timezone-aware objects to represent datetimes in UTC: datetime.datetime.now(datetime.UTC).
[2025-09-13T17:55:27.282+0000] {subprocess.py:93} INFO -   today = datetime.utcnow().strftime("%Y-%m-%d")
[2025-09-13T17:55:27.309+0000] {subprocess.py:93} INFO - + Text report saved to /opt/***/data/reports/report_2025-09-13.txt
[2025-09-13T17:55:27.475+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-09-13T17:55:27.613+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-13T17:55:27.614+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=currency_etl, task_id=analyze, run_id=manual__2025-09-13T10:27:29.902252+00:00, execution_date=20250913T102729, start_date=20250913T175523, end_date=20250913T175527
[2025-09-13T17:55:27.679+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-13T17:55:27.770+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-13T17:55:27.774+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
